modules/Spots/controller.php

```diff
@@ -994,9 +1007,9 @@ public function action_createQuotesSpotsData($filepath)
         while ($row = $db->fetchByAssoc($result)) {
             $x = new stdClass();
             $x->{$mod_strings['LBL_AN_QUOTES_OPPORTUNITY_NAME']} = $row['opportunityName'];
-            $x->{$mod_strings['LBL_AN_QUOTES_OPPORTUNITY_TYPE']} = $row['opportunityType'];
-            $x->{$mod_strings['LBL_AN_QUOTES_OPPORTUNITY_LEAD_SOURCE']} = $row['opportunityLeadSource'];
-            $x->{$mod_strings['LBL_AN_QUOTES_OPPORTUNITY_SALES_STAGE']} = $row['opportunitySalesStage'];
+            $x->{$mod_strings['LBL_AN_QUOTES_OPPORTUNITY_TYPE']} = (isset($app_list_strings['opportunity_type_dom'][$row['opportunityType']])?($app_list_strings['opportunity_type_dom'][$row['opportunityType']]):($row['opportunityType']));
```

Comment:
`$app_list_strings` seems undefined to me. Did you miss something?

Reply:
Definitely, yes.. I missed some `global $app_list_strings` .. fixed, now.